<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">

<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
  <title>Download Microme Software | Microme</title>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="shortcut icon" href="../sites/microme.eu/themes/custom/microme_fusion/favicon.ico" type="image/x-icon" />
  <link type="text/css" rel="stylesheet" media="all" href="../modules/node/node-O.css" />
<link type="text/css" rel="stylesheet" media="all" href="../modules/system/defaults-O.css" />
<link type="text/css" rel="stylesheet" media="all" href="../modules/system/system-O.css" />
<link type="text/css" rel="stylesheet" media="all" href="../modules/system/system-menus-O.css" />
<link type="text/css" rel="stylesheet" media="all" href="../modules/user/user-O.css" />
<link type="text/css" rel="stylesheet" media="all" href="../sites/microme.eu/modules/contrib/cck/theme/content-module-O.css" />
<link type="text/css" rel="stylesheet" media="all" href="../sites/microme.eu/modules/contrib/ctools/css/ctools-O.css" />
<link type="text/css" rel="stylesheet" media="all" href="../sites/microme.eu/modules/contrib/date/date-O.css" />
<link type="text/css" rel="stylesheet" media="all" href="../sites/microme.eu/modules/contrib/date/date_popup/themes/jquery.timeentry-O.css" />
<link type="text/css" rel="stylesheet" media="all" href="../sites/microme.eu/modules/contrib/fancy_login/css/fancy_login-O.css" />
<link type="text/css" rel="stylesheet" media="all" href="../sites/microme.eu/modules/contrib/filefield/filefield-O.css" />
<link type="text/css" rel="stylesheet" media="all" href="../sites/microme.eu/modules/contrib/fivestar/css/fivestar-O.css" />
<link type="text/css" rel="stylesheet" media="all" href="../sites/microme.eu/modules/contrib/genpass/genpass-O.css" />
<link type="text/css" rel="stylesheet" media="all" href="../sites/microme.eu/modules/contrib/panels/css/panels-O.css" />
<link type="text/css" rel="stylesheet" media="all" href="../sites/microme.eu/modules/contrib/quote/quote-O.css" />
<link type="text/css" rel="stylesheet" media="all" href="../modules/forum/forum-O.css" />
<link type="text/css" rel="stylesheet" media="all" href="../sites/microme.eu/modules/contrib/cck/modules/fieldgroup/fieldgroup-O.css" />
<link type="text/css" rel="stylesheet" media="all" href="../sites/microme.eu/modules/contrib/views/css/views-O.css" />
<link type="text/css" rel="stylesheet" media="all" href="../sites/microme.eu/themes/contrib/fusion/fusion_core/css/style-O.css" />
<link type="text/css" rel="stylesheet" media="all" href="../sites/microme.eu/themes/contrib/fusion/fusion_core/css/typography-O.css" />
<link type="text/css" rel="stylesheet" media="all" href="../sites/microme.eu/themes/contrib/fusion/fusion_core/css/superfish-O.css" />
<link type="text/css" rel="stylesheet" media="all" href="../sites/microme.eu/themes/contrib/fusion/fusion_core/css/superfish-navbar-O.css" />
<link type="text/css" rel="stylesheet" media="all" href="../sites/microme.eu/themes/contrib/fusion/fusion_core/css/superfish-vertical-O.css" />
<link type="text/css" rel="stylesheet" media="all" href="../sites/microme.eu/themes/custom/microme_fusion/css/microme-fusion-style-O.css" />
  <link type="text/css" rel="stylesheet" media="all" href="../sites/microme.eu/themes/contrib/fusion/fusion_core/css/grid16-960-O.css"/>
  <!--[if IE 8]>
  <link type="text/css" rel="stylesheet" media="all" href="../sites/microme.eu/themes/contrib/fusion/fusion_core/css/ie8-fixes-O.css" />
  <![endif]-->
  <!--[if IE 7]>
  <link type="text/css" rel="stylesheet" media="all" href="../sites/microme.eu/themes/contrib/fusion/fusion_core/css/ie7-fixes-O.css" />
  <![endif]-->
  <!--[if lte IE 6]>
  <link type="text/css" rel="stylesheet" media="all" href="../sites/microme.eu/themes/contrib/fusion/fusion_core/css/ie6-fixes-O.css"/>
<link type="text/css" rel="stylesheet" media="all" href="../sites/microme.eu/themes/custom/microme_fusion/css/ie6-fixes-O.css"/>
  <![endif]-->
    <script type="text/javascript" src="../misc/jquery.js?O"></script>
<script type="text/javascript" src="../misc/drupal.js?O"></script>
<script type="text/javascript" src="../sites/microme.eu/modules/contrib/fancy_login/js/fancy_login.js?O"></script>
<script type="text/javascript" src="../sites/microme.eu/modules/contrib/fivestar/js/fivestar.js?O"></script>
<script type="text/javascript" src="../sites/microme.eu/modules/contrib/panels/js/panels.js?O"></script>
<script type="text/javascript" src="../sites/microme.eu/modules/contrib/poormanscron/poormanscron.js?O"></script>
<script type="text/javascript" src="../sites/microme.eu/libraries/tinymce/jscripts/tiny_mce/plugins/media/js/embed.js?O"></script>
<script type="text/javascript" src="../sites/microme.eu/themes/contrib/fusion/fusion_core/js/jquery.bgiframe.min.js?O"></script>
<script type="text/javascript" src="../sites/microme.eu/themes/contrib/fusion/fusion_core/js/hoverIntent.js?O"></script>
<script type="text/javascript" src="../sites/microme.eu/themes/contrib/fusion/fusion_core/js/supposition.js?O"></script>
<script type="text/javascript" src="../sites/microme.eu/themes/contrib/fusion/fusion_core/js/supersubs.js?O"></script>
<script type="text/javascript" src="../sites/microme.eu/themes/contrib/fusion/fusion_core/js/superfish.js?O"></script>
<script type="text/javascript" src="../sites/microme.eu/themes/contrib/fusion/fusion_core/js/script.js?O"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, { "basePath": "/", "fancyLogin": { "screenFadeColor": "white", "screenFadeZIndex": "10", "loginBoxBackgroundColor": "white", "loginBoxTextColor": "black", "loginBoxBorderColor": "black", "loginBoxBorderWidth": "3px", "loginBoxBorderStyle": "solid", "loginPath": "user/login", "dimFadeSpeed": 500, "boxFadeSpeed": 1000, "hideObjects": 0, "disableRegistration": 0 }, "fivestar": { "titleUser": "Your rating: ", "titleAverage": "Average: ", "feedbackSavingVote": "Saving your vote...", "feedbackVoteSaved": "Your vote has been saved.", "feedbackDeletingVote": "Deleting your vote...", "feedbackVoteDeleted": "Your vote has been deleted." }, "cron": { "basePath": "/poormanscron", "runNext": 1471882269 } });
//--><!]]>
</script>
</head>

<body id="pid-download-microme-software" class="not-front not-logged-in page-node node-type-page no-sidebars full-node layout-main sidebars-split font-size-12 grid-type-960 grid-width-16">
  <div id="page" class="page">
    <div id="page-inner" class="page-inner">
      <div id="skip">
        <a href="#main-content-area">Skip to Main Content Area</a>
      </div>

      <!-- header-top row: width = grid_width -->
      
      <!-- header-group row: width = grid_width -->
      <div id="header-group-wrapper" class="header-group-wrapper full-width">
        <div id="header-group" class="header-group row grid16-16">
          <div id="header-group-inner" class="header-group-inner inner clearfix">
            <div id="secondary-menu" class="secondary-menu block">
<div id="secondary-menu-inner" class="secondary-menu-inner inner clearfix">
<ul class="links"><li class="menu-3395 first last"><a href="../user/login/index.html" title="Project members only">Log in</a></li>
</ul></div><!-- /secondary-menu-inner -->
</div><!-- /secondary-menu -->
            
                        <div id="header-site-info" class="header-site-info block">
              <div id="header-site-info-inner" class="header-site-info-inner inner">
                                <div id="logo">
                  <a href="../index.html" title="Home"><img src="../sites/microme.eu/themes/custom/microme_fusion/logo.png" alt="Home" /></a>
                </div>
                                              </div><!-- /header-site-info-inner -->
            </div><!-- /header-site-info -->
            
                        <div id="primary-menu" class="primary-menu block">
<div id="primary-menu-inner" class="primary-menu-inner inner clearfix">
<ul class="menu sf-menu"><li class="leaf first"><a href="../index.html" title="Home">Home</a></li>
<li class="expanded nolink-li"><span class="nolink">Browse</span><ul class="menu"><li class="expanded first nolink-li"><span class="nolink">Genomes</span><ul class="menu"><li class="leaf first"><a href="http://bacteria.microme.eu/index.html" title="An Ensembl Genome browser for the display and analyses of the 24 bacterial species in our priority list. Gene entries are cross-referenced to pathways stored in the bacterial Reactome knowledge-base">Ensembl Bacteria</a></li>
<li class="leaf last"><a href="http://www.genoscope.cns.fr/agc/microscope" title="A platform for microbial comparative genome analysis and manual functional annotation">Microscope</a></li>
</ul></li>
<li class="expanded last nolink-li"><span class="nolink">Pathways</span><ul class="menu"><li class="leaf first"><a href="http://pathway.microme.eu" title="An interactive visual interface to the Microme database of metabolic pathways and processes">Bacterial Reactome</a></li>
<li class="leaf last"><a href="http://www.unipathway.org" title="A fully manually curated resource for the representation and annotation of metabolic pathways, with data extensively cross-linked to existing pathway and sequence resources">UniPathway</a></li>
</ul></li>
</ul></li>
<li class="expanded nolink-li"><span class="nolink">Tools</span><ul class="menu"><li class="leaf first"><a href="../mart/index.html" title="Explore complete sets of reactions inferred over thousands of genomes">Genome-Reaction Data Warehouse</a></li>
<li class="leaf"><a href="http://rsat.bigre.ulb.ac.be/rsat/index_neat.html" title="The Network Analysis Tools (NeAT) provide a series of modular computer programs specifically designed for the analysis of biological networks">Network Analysis</a></li>
<li class="leaf"><a href="http://tebacten.bioinfo.cnio.es" title="TeBactEn (text mining for bacterial enzymes) is a tool for retrieval, extraction and annotation of bacterial enzymatic reactions and pathways from the literature">Text Mining</a></li>
<li class="expanded nolink-li"><span class="nolink">Web Services</span><ul class="menu"><li class="leaf first"><a href="../bacterial-reactome-web-services/index.html" title="Fetch events and molecules from the bacterial Reactome knowledge-base using the Reactome WS API">Reactome</a></li>
<li class="leaf"><a href="../microcyc-web-services/index.html" title="Extract genomic and metabolic data from MicroCyc by using web services">MicroCyc</a></li>
<li class="leaf last"><a href="../rsatneat-web-services/index.html" title="The Regulatory Sequence Analysis Tools and Network Analysis Tools are also available as web services">RSAT/NeAT</a></li>
</ul></li>
<li class="leaf"><a href="http://microme.systemsbiology.nl/" title="Fluxome">Fluxome</a></li>
<li class="leaf last"><a href="http://botero.cnb.csic.es/Retrobiosynthesis/" title="">Botero Tool for Retrobiosynthesis</a></li>
</ul></li>
<li class="expanded active-trail nolink-li"><span class="nolink">Download</span><ul class="menu"><li class="leaf first"><a href="../download-microme-data/index.html" title="Download Microme Data">Data</a></li>
<li class="leaf last active-trail"><a href="index.html" title="Download Microme Software" class="active">Software</a></li>
</ul></li>
<li class="collapsed nolink-li"><span class="nolink">Members Area</span></li>
<li class="expanded nolink-li"><span class="nolink">About</span><ul class="menu"><li class="leaf first"><a href="../organization/index.html" title="">Overview</a></li>
<li class="leaf"><a href="../members/index.html" title="">Project Partners</a></li>
<li class="expanded last nolink-li"><span class="nolink">Documentation</span><ul class="menu"><li class="leaf first"><a href="../Publications/index.html">Publications</a></li>
<li class="leaf"><a href="../pages/microme-sops/index.html" title="">SOPs</a></li>
<li class="leaf"><a href="../training/index.html" title="Training">Training</a></li>
<li class="leaf last"><a href="http://www.ebi.ac.uk/milestones" title="">Milestones</a></li>
</ul></li>
</ul></li>
<li class="leaf last"><a href="../contact-us/index.html" title="Contact us">Contact us</a></li>
</ul></div><!-- /primary-menu-inner -->
</div><!-- /primary-menu -->
          </div><!-- /header-group-inner -->
        </div><!-- /header-group -->
      </div><!-- /header-group-wrapper -->

      <!-- preface-top row: width = grid_width -->
      
      <!-- main row: width = grid_width -->
      <div id="main-wrapper" class="main-wrapper full-width">
        <div id="main" class="main row grid16-16">
          <div id="main-inner" class="main-inner inner clearfix">
            
            <!-- main group: width = grid_width - sidebar_first_width -->
            <div id="main-group" class="main-group row nested grid16-16">
              <div id="main-group-inner" class="main-group-inner inner">
                
                <div id="main-content" class="main-content row nested">
                  <div id="main-content-inner" class="main-content-inner inner">
                    <!-- content group: width = grid_width - (sidebar_first_width + sidebar_last_width) -->
                    <div id="content-group" class="content-group row nested grid16-16">
                      <div id="content-group-inner" class="content-group-inner inner">
                        <div id="breadcrumbs" class="breadcrumbs block">
<div id="breadcrumbs-inner" class="breadcrumbs-inner inner clearfix">
<div class="breadcrumb"><a href="../index.html">Home</a></div></div><!-- /breadcrumbs-inner -->
</div><!-- /breadcrumbs -->
                      
                      
                        
                        <div id="content-region" class="content-region row nested">
                          <div id="content-region-inner" class="content-region-inner inner">
                            <a id="main-content-area"></a>
                                                        <div id="content-inner" class="content-inner block">
                              <div id="content-inner-inner" class="content-inner-inner inner">
                                                                <h1 class="title">Download Microme Software</h1>
                                                                                                <div id="content-content" class="content-content">
                                  
<div id="node-266" class="node odd full-node node-type-page">
  <div class="inner">
    
    
    
    
    <div class="content clearfix">
      
<p style="text-align: justify;">Our FTP site<strong><a href="ftp://ftp.ebi.ac.uk/pub/databases/microme/software">&nbsp;</a></strong>hosts some software which can be used by external users to access and analyse various elements of the Microme data set.</p>
<p style="text-align: justify;"><strong><br />
</strong></p>
<p style="text-align: justify;"><strong>Reaction Matrix</strong></p>
<p style="text-align: justify;">We provide a <strong><a href="ftp://ftp.ebi.ac.uk/pub/databases/microme/software/reaction_matrix">Perl script and accompanying data</a></strong> designed to allow users to analyse the spread of inferred reactions using the large Microme data set (reaction matrix): find pathways in genomes specified by their parent node in taxonomy, reactions in RhEA involved in these pathways, and the genes assigned to UniProt proteins associated with these reactions.</p>
<p style="text-align: justify;">&nbsp;</p>
<p style="text-align: justify;"><strong>Genome-scale functional characterization of a species</strong></p>
<p style="text-align: justify;"><strong><br />
</strong>We provide a <strong><a href="ftp://ftp.ebi.ac.uk/pub/databases/microme/software/genome_scale_functional_analyses">software package</a></strong> that performs the following functionalities:</p>
<ul>
 <li>find an optimal media (OM) for a given organism - an OM is the media in terms of uptake reactions that still enables maximal growth rate (the latter refers to the limit growth rate obtained at rich media)</li>
 <li>fing a minimal media for a given organism, when the organism is forced to grow at list X% of its maximal biomass</li>
 <li>given an organism, find essential genes, i.e. whose knock-out results in zero growth</li>
 <li>similarly, find essential gene pairs; the percentage of essential genes and gene pairs provides a measure of the network's genetic robustness</li>
</ul>
<p style="text-align: justify;"><strong><br />
Biolog</strong></p>
<p style="text-align: justify;">The Phenotype MicroArray (OmniLog®) system is able to simultaneously capture a large number of phenotypes by recording an organism's respiration over time on distinct substrates. The phenotypic reaction of single-celled organisms and animal cell cultures to up to 2,000 environmental challenges can be recorded on sets of 96-well microtiter plates.</p>
<p style="text-align: justify;">One of our project members, DSMZ – German Collection of Microorganisms and Cell Cultures in Braunschweig, has developed an application-oriented software package for the analysis of Biolog high-throughput phenotypic data. The <a href="http://cran.r-project.org/web/packages/opm/index.html">opm</a> (OmniLog Phenotype MicroArray) package, which is based on the free software environment R, allows comprehensive statistical analyses on quantitative data of energy production curves and a better graphic presentation.</p>
<p style="text-align: right;"><a href="http://www.dsmz.de/research/microorganisms/projects/analysis-of-omnilog-phenotype-microarray-data.html">Read More</a></p>
<p style="text-align: justify;"><strong><br />
</strong></p>
<p style="text-align: justify;"><strong>Web Services</strong></p>
<p style="text-align: justify;">We provide a set of web services example clients, developed in different languages, for exploring the data provided by some partners of the project.</p>
<table border="1" style="margin: auto;">
 <tbody>
 <tr>
  <td><em><strong>Name</strong></em></td>
  <td style="text-align: center;"><em><strong>Server</strong></em></td>
  <td style="text-align: center;"><em><strong>Language</strong></em></td>
  <td style="text-align: center;"><em><strong>Description</strong></em></td>
 </tr>
 <tr>
  <td><a href="ftp://ftp.ebi.ac.uk/pub/databases/microme/software/ws_clients/java/reactome/ReactomeWSClient.jar" class="file">ReactomeWSClient.jar</a></td>
  <td style="text-align: center;">Reactome</td>
  <td style="text-align: center;">Java</td>
  <td>
  <p>Examples of querying the bacterial Reactome repository using the WS Java API</p>
  </td>
 </tr>
 <tr>
  <td><span class="file"><a href="ftp://ftp.ebi.ac.uk/pub/databases/microme/software/ws_clients/perl/reactome/reactome_ws_client_example.pl" class="file">reactome_ws_client_example.pl</a></span></td>
  <td style="text-align: center;">Reactome</td>
  <td style="text-align: center;">Perl</td>
  <td>
  <p>Export pathway diagrams to SVG/BioPax and other simple examples</p>
  </td>
 </tr>
 <tr>
  <td>
  <p><span class="file"><a href="ftp://ftp.ebi.ac.uk/pub/databases/microme/software/ws_clients/perl/rsat-neat/infer-operons_client.pl" class="file">infer-operons_client.pl</a>*<br />
  </span></p>
  </td>
  <td style="text-align: center;">RSAT/NeAT</td>
  <td style="text-align: center;">Perl</td>
  <td style="text-align: justify;">Predict operons for a given organism (either restricted to a list of query genes, or applied to all genomes of the genome)</td>
 </tr>
 <tr>
  <td>
  <p><span class="file"><a href="ftp://ftp.ebi.ac.uk/pub/databases/microme/software/ws_clients/java/microcyc/microcyc-webserviceclient-1.0-20120319-1410-project.zip" class="file">microcyc-webserviceclient-1.0-20120319-1410-project.zip</a>**<br />
  </span></p>
  </td>
  <td style="text-align: center;">&nbsp;Microscope</td>
  <td style="text-align: center;">Java</td>
  <td>Java template project (Eclipse) showing how to query the MicroCyc repository</td>
 </tr>
 <tr>
  <td>
  <p><a href="ftp://ftp.ebi.ac.uk/pub/databases/microme/software/ws_clients/perl/microcyc/get-gpr-genoscope.pl">get-gpr-genoscope.pl</a></p>
  </td>
  <td style="text-align: center;">&nbsp;Microscope</td>
  <td style="text-align: center;">Perl</td>
  <td>Retrieve the list of GPRs</td>
 </tr>
 <tr>
  <td>
  <p><a href="ftp://ftp.ebi.ac.uk/pub/databases/microme/software/ws_clients/perl/microcyc/get-organisms-genoscope.pl">get-organisms-genoscope.pl</a></p>
  </td>
  <td style="text-align: center;">Microscope</td>
  <td style="text-align: center;">Perl</td>
  <td>Get the list of supported organisms</td>
 </tr>
 <tr>
  <td>
  <p><a href="ftp://ftp.ebi.ac.uk/pub/databases/microme/software/ws_clients/python/microcyc/supported-organisms-genoscope.py">supported-organisms-genoscope.py</a></p>
  </td>
  <td style="text-align: center;">Microscope</td>
  <td style="text-align: center;">Python</td>
  <td>Get the list of supported organisms</td>
 </tr>
 <tr>
  <td><a href="ftp://ftp.ebi.ac.uk/pub/databases/microme/software/ws_clients/python/microcyc/gene-ec-genoscope.py">gene-ec-genoscope.py</a></td>
  <td style="text-align: center;">Microscope</td>
  <td style="text-align: center;">Python</td>
  <td style="text-align: justify;">Get the list of genes, the taxonnomic ID and the species name of the specified organism and EC number of GPR</td>
 </tr>
 <tr>
  <td><a href="ftp://ftp.ebi.ac.uk/pub/databases/microme/software/ws_clients/python/microcyc/metabolic-network-genoscope.py">metabolic-network-genoscope.py</a></td>
  <td style="text-align: center;">Microscope</td>
  <td style="text-align: center;">Python</td>
  <td style="text-align: justify;">Get the list of reactions and compounds, the list of arcs (list of compounds involved in each reaction)</td>
 </tr>
 </tbody>
</table>
<p>(*) For other, more comprehensive examples of querying RSAT/NeAT, go to <a href="http://rsat.ulb.ac.be/rsat/ws_clients.html">http://rsat.ulb.ac.be/rsat/ws_clients.html</a></p>
<p>(**) The project is also available at <a href="http://www.genoscope.cns.fr/microme/microcyc-wsc/index.html">http://www.genoscope.cns.fr/microme/microcyc-wsc/index.html</a></p>
    </div>

    
      </div><!-- /inner -->

  </div><!-- /node-266 -->
                                                                  </div><!-- /content-content -->
                                                              </div><!-- /content-inner-inner -->
                            </div><!-- /content-inner -->
                          </div><!-- /content-region-inner -->
                        </div><!-- /content-region -->

                                              </div><!-- /content-group-inner -->
                    </div><!-- /content-group -->

                                      </div><!-- /main-content-inner -->
                </div><!-- /main-content -->

                              </div><!-- /main-group-inner -->
            </div><!-- /main-group -->
          </div><!-- /main-inner -->
        </div><!-- /main -->
      </div><!-- /main-wrapper -->

      <!-- postscript-bottom row: width = grid_width -->
      
      <!-- footer row: width = grid_width -->
      <div id="footer-wrapper" class="footer-wrapper full-width">
<div id="footer" class="footer row grid16-16">
<div id="footer-inner" class="footer-inner inner clearfix">
  
<div id="block-block-10" class="block block-block odd first last grid16-16">
  <div class="inner clearfix">
            <div class="content clearfix">
      
<p style="text-align: center;"><a href="http://www.ebi.ac.uk"><img src="../system/files/images/ebi_footer_logo.png" height="70" width="158" alt="image" /></a><a href="http://cordis.europa.eu/fp7/dc/index.cfm"><img src="../system/files/images/footer_bg_0.png" height="70" width="180" alt="image" /></a></p>
<p style="text-align: center;"><a href="../contact-webmaster/index.html">Contact the webmaster</a></p>
    </div>
  </div><!-- /block-inner -->
</div><!-- /block -->
</div><!-- /footer-inner -->
</div><!-- /footer -->
</div><!-- /footer-wrapper -->

      <!-- footer-message row: width = grid_width -->
      <div id="footer-message-wrapper" class="footer-message-wrapper full-width">
        <div id="footer-message" class="footer-message row grid16-16">
          <div id="footer-message-inner" class="footer-message-inner inner clearfix">
            <div id="footer-message-text" class="footer-message-text block">
<div id="footer-message-text-inner" class="footer-message-text-inner inner clearfix">
MICROME is an EU Framework Programme 7 Collaborative Project. Grant Agreement Number 222886-2<br />
Site maintained by the External Services team at <a href="http://www.ebi.ac.uk/">EMBL-EBI</a> | <a href="http://www.ebi.ac.uk/Information/termsofuse.html">Terms of Use</a> | <a href="http://www.ebi.ac.uk/Information/privacy.html">Privacy</a> | <a href="http://www.ebi.ac.uk/Information/e-directive.html">Cookies</a></div><!-- /footer-message-text-inner -->
</div><!-- /footer-message-text -->
          </div><!-- /footer-message-inner -->
        </div><!-- /footer-message -->
      </div><!-- /footer-message-wrapper -->

    </div><!-- /page-inner -->
  </div><!-- /page -->
  <div id="fancy_login_dim_screen"></div><div id="fancy_login_login_box"><div id="fancy_login_form_contents"><a href="#" id="fancy_login_close_button">X</a><div id="fancy_login_user_login_block_wrapper"><form action="/download-microme-software?destination=node%2F266"  accept-charset="UTF-8" method="post" id="user-login-form">
<div><div class="form-item" id="edit-name-wrapper">
 <label for="edit-name">Username: <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" maxlength="60" name="name" id="edit-name" size="15" value="" class="form-text required" />
</div>
<div class="form-item" id="edit-pass-wrapper">
 <label for="edit-pass">Password: <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" name="pass" id="edit-pass"  maxlength="60"  size="15"  class="form-text required" />
</div>
<div class="item-list"><ul><li class="first last"><a href="../user/password/index.html" title="Request new password via e-mail.">Request new password</a></li>
</ul></div><input type="hidden" name="form_build_id" id="form-2kONAn36GNt9aN8T2IE8z1tbghApF6pMXiRD0vJwfso" value="form-2kONAn36GNt9aN8T2IE8z1tbghApF6pMXiRD0vJwfso"  />
<input type="hidden" name="form_id" id="edit-fancy-login-user-login-block" value="fancy_login_user_login_block"  />
<input type="submit" name="op" id="edit-submit-1471878694" value="Log in"  class="form-submit" />

</div></form>
</div></div></div></body>
</html>

<!-- Localized -->